.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,23 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v2
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
we'll have to create that token and put it into the dependabot secrets, correct? It's not one available automatically?

Reply:
actually not! it's the token that's created by default by GitHub itself for each run.
And with the `permissions` set to write earlier, it also got the permissions to modify the PR, in this case it allows the token to merge it.

The file as it's here is working since two years over at the other repository I've linked. I also verified that there's no additional token in the repository config over there, so this workflow should "just work" (tm) ^^